bot.py

```diff
@@ -552,7 +553,9 @@ async def lab_hours_reminder():
             if special_hours:
                 description += f'\n**Special Lab Hours!**:\n{special_hours}'
             embed = discord.Embed(title=title, description=description, color=color)
-            await lab_channel.send(embed=embed)
+
+            if lab_channel:
```

Comment:
This conditional is not necessary, on line 537 method returns if `lab_channel` is invalid